socorro/external/es/super_search_fields.py

```diff
@@ -176,39 +171,71 @@ def update_field(self, **kwargs):
                 'be created before it can be updated. ' % params['name']
```

Comment:
``params['name']`` is the name of the field (so something like "signature" or "product") and is also used as the ID of the field, since it is unique.